ui/app/registration/services/patientService.js

```diff
@@ -29,6 +29,23 @@ angular.module('bahmni.registration')
             };
             return patientServiceStrategy.search(config);
         };
+        var searchDuplicatePatients = function (query, gender, birthDate, contactNumber) {
+            var config = {
+                params: {
+                    q: query,
+                    gender: gender,
+                    birthDate: birthDate,
+                    addressFieldName: "address4",
```

Comment:
@IipperMangesh sub-division is mapped to address3 
![image](https://user-images.githubusercontent.com/22587098/53811424-06816900-3f62-11e9-950f-959f4ca569b8.png)

Reply:
Yes, done these changes.